src/core/annotation.js

```diff
@@ -922,12 +922,16 @@ class WidgetAnnotation extends Annotation {
 
     const localResources = getInheritableProperty({ dict, key: "DR" });
     const acroFormResources = params.acroForm.get("DR");
+    const appearanceResources =
+      this.appearance && this.appearance.dict.get("Resources");
+
     this._fieldResources = {
       localResources,
       acroFormResources,
+      appearanceResources,
       mergedResources: Dict.merge({
         xref: params.xref,
-        dictArray: [localResources, acroFormResources],
+        dictArray: [localResources, acroFormResources, appearanceResources],
```

Comment:
Just a small questions here: Do you want this ordering, since that'd prefer the `appearanceResources` *last* when merging the dictionaries, since `Dict.merge` iterates through this array left-to-right?
Or would swapping the order of `acroFormResources` and `appearanceResources` here make sense?

It probably doesn't matter, since this situation probably is an an edge-case anyway, but figured I'd ask anyway :-)

Reply:
Probably a edge case, but I think it's better to swap.